Civi/API/Subscriber/PermissionCheck.php

```diff
@@ -87,4 +87,26 @@ public function onApiAuthorize(\Civi\API\Event\AuthorizeEvent $event) {
     }
   }
 
+  /**
+   * check API for ACL permission.
+   * @param $apiRequest
```

Comment:
This is really trivial so you don't need to submit a new commit just for this- but if we were enforcing more of the PHPCS rules then the comment block would look like:

  /**
  *  Check API for ACL permission.  
  *
  * @param array $apiRequest
  * 
  * @ return bool
  */

ie. cap in first word & line breaks around the @param declarations. (also $apiRequest is an array)